### ISSUE TITLE
Use in-memory FS or buffer instead of /tmp WorkDir

### ISSUE THREAD

**Issue by VojtechVitek** [goware/saml#6]
https://github.com/goware/saml/blob/master/files.go#L11

https://github.com/spf13/afero
https://github.com/blang/vfs
or

**Comment by VojtechVitek** [comment]
I see, the xmlsec1 actually picks up the certs from the FileSystem..

Hmm.. what can we do to avoid writing into the disk?

**Comment by xiam** [comment]
Yeah, this is because we need xmlsec1 to read certain things from disk. In order to not write things to disk an user can mount a small RAM filesystem, such as tmpfs.

**Comment by VojtechVitek** [comment]
@xiam I'm really hoping we could replace xmlsec1 binary with a pure Go package one day.
